src/components/InlineForm.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useMemo, useReducer, useRef, useState } from 'react';
+import styled from 'styled-components';
+import { IconQuestionCircleOutline } from '@taskany/icons';
+
+import { KeyCode, useKeyboard } from '../hooks/useKeyboard';
+import { useClickOutside } from '../hooks/useClickOutside';
+import { nullable } from '../utils';
+
+import { Form } from './Form';
+import { Popup } from './Popup';
+
+interface RenderTriggerProps {
+    onClick: () => void;
+}
+
+interface InlineFormProps {
+    renderTrigger?: (props: RenderTriggerProps) => React.ReactNode;
+    onSubmit: () => void;
+    children: React.ReactNode;
+    tip?: React.ReactNode;
+    onReset: () => void;
+    isSubmitted?: boolean;
```

Comment:
What the reason to have this prop? Why onSubmit callback is not enough?

Reply:
ok, onSubmit callback is enough